Assignments/SCF-Session-2/hexadecimalCalc/HexCalc.java

```diff
@@ -0,0 +1,277 @@
+package hexadecimalCalc;
+
+import java.util.Scanner;
+
+class Functions
+{
+	public String add(String hexNum1, String hexNum2)
+	{
+		int decNum1 = hexToDec(hexNum1);
```

Comment:
Naming convention. `hexNum1`, `hexNum2`, `decNum2`, `decNum1`. Please use the relevant names like `firstHexNumber`, `secondDecimalNumber`.

Reply:
done